pages/what-do-you-want.tsx

```diff
@@ -45,9 +67,7 @@ export default function WhatDoYouWant(): JSX.Element {
             <Button type="button" outline onClick={() => router.back()}>
               Back
             </Button>
-            <Button type="submit" onClick={handleNext}>
-              Next
-            </Button>
+            <Button type="submit">Next</Button>
```

Comment:
Since the user is navigated to the next page immediately after clicking the radio button, I think we don't need the Next button anymore.

Reply:
should the options still be radio buttons, or just a clickable div?

Reply:
it's ok to keep the radio buttons

Reply:
done